### ISSUE TITLE
Update PyPI release

### ISSUE THREAD

**Issue by sneakers-the-rat** [datajoint/element-session#18]
Hello! Looks like the pre-release version is about a year old, and the version in the repo is still 0.1.0b0 despite substantial changes since then. Wondering if there's a particular reason the PyPI version isn't being kept synchronized, or if I should just pin a commit in the git repository rather than by package version in downstream packages?

**Comment by kabilar** [comment]
Thanks for the report @sneakers-the-rat.

We just went through a development sprint.  We will be updating the version, tagging, and releasing these changes to pypi by the end of May.  For the interim, please install from source:

```
pip install git+https://github.com/datajoint/element-event.git
```

Install a specific commit:
```
pip install git+https://github.com/datajoint/element-event.git@<Enter commit hash>
```

**Comment by kabilar** [comment]
Hi @sneakers-the-rat, we just published a new release (`0.1.2`) of this package.  Please find it on [PyPI](https://pypi.org/project/element-session).  Thanks.

**Comment by sneakers-the-rat** [comment]
y'all rock, thanks
